### ISSUE TITLE
Question: where saves the "save" button the Export?

### ISSUE THREAD

**Issue by Espo02** [NewEconomyMovement/NEMiOSApp#200]
Just a question where the NEMiOSAPP saves the QRcode if I hit the "speichern" Button?
There is no picture in my camare roll and there is no picture in my clipboard. 
And a nother Question. Why should I send my wallet with XEM via TWITTERin the world? 
I'm not using twitter a lot, what do I not seeing?

![photo-2016-04-15-18-08-50_1474](https://cloud.githubusercontent.com/assets/17567386/14567651/70ee6624-0335-11e6-9aab-662a47e63465.PNG)

![photo-2016-04-15-18-03-42_1473](https://cloud.githubusercontent.com/assets/17567386/14567766/f43bdfac-0335-11e6-9888-f26dd2cdd2ca.PNG)

**Comment by Espo02** [comment]
PS: the close "X" is very hard to find ;)

**Comment by jabo38** [comment]
It should save the QR to your camera roll.  It works on my iphone6.  Can you confirm it doesn't save to your camera roll?  Also check to see if in your settings if the app has access to your camera roll. 
![image](https://cloud.githubusercontent.com/assets/10395907/14663403/ad7bcb58-06fb-11e6-855d-c9e0223e08e2.png)

**Comment by Espo02** [comment]
Ah, Ok. Yes This right was missing. Now it works.
Can the NemApp show an error Message to the user if he wants to save the qr code and the app hasn't the right?
Annd... Can the name of the account be drawn over the QR Code? that would help if you export more than one account,

**Comment by jabo38** [comment]
@kmiliak how hard is it to get the name of the account included with the QR?

And is it possible to get a better error message for this problem?

**Comment by fyahfox** [comment]
?

**Comment by kmiliak** [comment]
@jabo38 we can include name to account it's 1 - 1.5 hour roughly.
As for error message, we can add pop up

**Comment by jabo38** [comment]
@kmiliak Please don't include the name then.  

But really Espo is right, there is no reason for Twitter to be included in there when sharing the QR.  I think we discussed this before?  It can be deleted if it won't cost extra.  

Please make the error a person if they aren't really saving to camera roll.  

Also, we have had numerous complaints about the "x" being impossible to see, which it is.  It would be nice if you could figure out a solution for that.  That also occurs on the My Info panel.  So people also complain about it there.

**Comment by fyahfox** [comment]
Yes, of course we need an error message for such an important functionality. I just didn't notice yet, but that is not debatable.

The "x" issue is something general and shouldn't be mixed with this issue.

**Comment by Espo02** [comment]
`Also, we have had numerous complaints about the "x" being impossible to see, which it is. It would be nice if you could figure out a solution for that. That also occurs on the My Info panel. So people also complain about it there. `

Is it possible to close the the "[MAIL][TWITTER]" screen by tapping somwhere in the grayed section [Background]?

**Comment by fyahfox** [comment]
@Espo02 Please create a new issue for this ;)

**Comment by Espo02** [comment]
The "save as" Button (the right one) Problem is solved very good. 
It now shows the normal iOS menu with all possible programs to use a picture.
![photo-2016-11-15-07-37-14_3940](https://cloud.githubusercontent.com/assets/17567386/20295667/8ffc98aa-ab06-11e6-8ea0-a3fe2430667e.PNG)

But the "Save" (left one) Button don't give a feedback that the Image was saved in the Camery roll.
The same as here: https://github.com/NemProject/NEMiOSApp/issues/277
